other_projects_that_use_sympy.tex

```diff
@@ -0,0 +1,38 @@
+\subsection{SymPy Gamma}\label{sympy-gamma}
+
+SymPy Gamma is a simple web application based on Google App Engine that 
+executes and displays the results of SymPy expressions as well as
+additional related computations, in a fashion similar to that of
+Wolfram\textbar{}Alpha. For instance, entering an integer will display
+prime factors, digits in the base-10 expansion, and a factorization
+diagram. Entering a function will give its docstring; in general,
+entering an arbitrary expression will provide the derivative, integral,
+series expansion, plot, and roots.
+
+SymPy Gamma also has several additional features than just computing the
+results using SymPy.
+
+\begin{itemize}
+\item
+  It displays integration steps, differentiation steps in detail, which
+  can be viewed through this
+  \href{http://www.sympygamma.com/input/?i=integrate\%281\%20/\%28\%28x\%2B1\%29\%28x\%2B3\%29\%28x\%2B5\%29\%29\%29}{Link}.
+\item
+  It also displays the factor tree diagrams for numbers, which is also
+  illustrated through this
+  \href{http://www.sympygamma.com/input/?i=112}{link}.
```

Comment:
Screenshots might be better here as they add permanency to the content of the paper.